packages/liveblocks-redux/src/index.ts

```diff
@@ -207,24 +214,33 @@ const internalEnhancer = <TState>(options: {
 
       const store = createStore(newReducer, initialState, enhancer);
 
-      function enterRoom(roomId: string) {
-        if (storageRoot) {
-          return;
+      function enterRoom(newRoomId: string): () => void {
+        if (lastRoomId === newRoomId) {
+          return lastLeaveFn!;
```

Comment:
would it make sense to make this consistent with core library and return `{ room, leave }` ?

Reply:
Don’t think it’s needed, since `enterRoom()` will just mutate the Zustane store, where `room` already exists. Maybe by the same argument, we could even let it return `void` like before, since you could just call `leaveRoom()` later. Internally it will keep a reference to the leave function already. We don’t have to expose it I think—thanks!

Reply:
Fixed in 3026a119eeba57c0f376d2178b5f5652255179f8 (in the other PR).